dcicutils/jh_utils.py

```diff
@@ -149,11 +149,20 @@ def find_valid_file_or_extra_file(obj_id, format):
     home_dir = '/home/jovyan'
     # hardcoded so that we can find which volume to look in by file type
     # will need to be updated if more File types are added to proc file bucket
-    if 'FileVistrack' in file_meta['@type'] or 'FileProcessed' in file_meta['@type']:
+    if file_meta['open_data_url']:
+        data_dir = 'open_data'
+        if 'FileVistrack' in file_meta['@type'] or 'FileProcessed' in file_meta['@type']:
+            file_type = 'wfoutput'
+        else:
+            file_type = 'files'
+        open_data_prefix = '{}'.format(file_type)
```

Comment:
Why not just call this `open_data_prefix` from the start? The string build here is unnecessary.

Reply:
Changing this to be 'fourfront-webprod/{}', that was why it was separate.